integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/container/ContainerSuite.java

```diff
@@ -150,6 +154,59 @@ public void startDorisContainer() {
     }
   }
 
+  public void startMySQLContainer(Class<?> clz) {
```

Comment:
Why do we need to pass a class instance to the method, can we use a string instead?

Reply:
Because I want to define the rule of database in one place,
if we let callers pass in string as DB name, they might forget how to name db correctly.

Reply:
> if we let callers pass in string as DB name, they might forget how to name db correctly.

This doesn't seem very elegant. What you need is just a database name that can be shared in different tests, am I right?

Reply:
I agree with you, I think create a enum contains all test database names might be a great choice, this ensure that caller won't pass in invalid database name.

e.g.

```
public enum TestDatabaseName {
    MYSQL_JdbcBackend,
    MYSQL_MySQLIT,
    POSTGRES,
}
```

Reply:
> I agree with you, I think create a enum contains all test database names might be a great choice, this ensure that caller won't pass in invalid database name.
> 
> e.g.
> 
> ```
> public enum TestDatabaseName {
>     MYSQL_JdbcBackend,
>     MYSQL_MySQLIT,
>     POSTGRES,
> }
> ```

I think it's better than the current one, if there's no better way, let's use this one.

Reply:
okay, I'll take time to implement it.